scripts/test/partitionSplitting/system-test-instance.properties

```diff
@@ -65,162 +65,21 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,IngestStack,PartitionSpl
-sleeper.compaction.task.cpu.architecture=X86_64
-
-# The CPU and memory for a compaction task. Which of these are used will depend on the chosen CPU architecture.
-# See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html for valid options.
-sleeper.compaction.task.arm.cpu=1024
-sleeper.compaction.task.arm.memory=4096
-sleeper.compaction.task.x86.cpu=1024
-sleeper.compaction.task.x86.memory=4096
-
-# Flag to enable/disable storage of tracking information for compaction jobs and tasks.
-sleeper.compaction.status.store.enabled=true
-
-# The time to live in seconds for compaction job updates in the status store. Default is 1 week
-sleeper.compaction.job.status.ttl=604800
-
-# The time to live in seconds for compaction task updates in the status store. Default is 1 week
-sleeper.compaction.task.status.ttl=604800
-
-# The retention period for CloudWatch logs. See https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html for valid options.
-# Use -1 to indicate infinite retention.
-sleeper.log.retention.days=30
-
-# The amount of memory in MB for the lambda that identifies partitions that need splitting
-sleeper.partition.splitting.finder.memory=2048
-
-# The timeout for the lambda that identifies partitions that need splitting in seconds
-sleeper.partition.splitting.finder.timeout.seconds=900
-
-# The amount of memory for the lambda that splits partitions
-sleeper.partition.splitting.memory=2048
-
-# The timeout for the lambda that splits partitions in seconds
-sleeper.partition.splitting.timeout.seconds=900
-
-# The maximum number of simultaneous connections to S3 from a single client
-sleeper.s3.max-connections=25
-
-# Fargate version
-sleeper.fargate.version=1.4.0
-
-# The frequency with which the WriteData class refreshes the list of partitions from the state store
-# (NB Refreshes only happen once a batch of data has been written so this is a lower bound
-# on the refresh frequency.)
-sleeper.ingest.partition.refresh.period=120
-
-# The frequency with which the lambda that checks whether ingest tasks need creating runs (in minutes).
-sleeper.ingest.task.creation.period.minutes=1
-
-# The number of files to read in a compaction job. Note that the state store
-# must support atomic updates for this many files. For the DynamoDBStateStore this
-# is 11.
-# (NB This does not apply to splitting jobs which will run even if there is only 1 file.)
-sleeper.compaction.files.batch.size=11
-
-# The way in which records are held in memory before they are written to a local store
-# Valid values are 'arraylist' and 'arrow'
-sleeper.ingest.record.batch.type=arrow
-
 # The way in which partition files are written to the main Sleeper store
 # Valid values are 'direct' (which writes using the s3a Hadoop file system) and 'async' (which writes locally and then
 # copies the completed Parquet file asynchronously into S3)
 sleeper.ingest.partition.file.writer.type=async
```

Comment:
Should this be made the default? It seems like we should be testing against the default, but we've used this value for some time.

Reply:
Yes, let's delete this.